### DISCUSSION TITLE
Derpfest issue sycn

### DISCUSSION THREAD

**Discussion by mozzaru** [ROM-builders/temporary#17628] [General]
![IMG_20230130_090829](https://user-images.githubusercontent.com/100791931/215372940-d9ec3ef9-13ec-41e3-9e3e-edf40c3d07af.jpg)
https://cirrus-ci.com/task/6485819846819840
![Screenshot_2023-01-30-08-48-09-08_da139dc6c40ce76fef24574850ad1949](https://user-images.githubusercontent.com/100791931/215371055-ff2bbcc5-ac3f-474a-a18c-9beb15f58ce1.jpg)

**Comment by Apon77** [comment]
Your local manifest not even cloning properly.

**Reply by mozzaru** [reply]
@Apon77 https://github.com/Mozzaru/local_manifest/blob/derp-13/local_manifest.xml

Previously, I didn't use the framework and any more but the sync still failed

tell me which one

**Reply by Apon77** [reply]
> @Apon77 https://github.com/Mozzaru/local_manifest/blob/derp-13/local_manifest.xml
> 
> Previously, I didn't use the framework and any more but the sync still failed
> 
> tell me which one

you are cloning evo branch but that does not exists in your local manifests

**Reply by mozzaru** [reply]
@Apon77 for derp what a problem?
